.github/workflows/tyk-pro.yaml

```diff
@@ -0,0 +1,47 @@
+name: Tyk Pro Push Helm
+on: push
+jobs:
+  push:
+    name: Tyk Pro Push CS
+    runs-on: ubuntu-latest
+    branches:
+      - main
+      - release-**
+    paths:
+      - 'tyk-pro/**'
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install Helm, create dirs
+        run: |
+          echo "Installing Helm..."
+          wget -q https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz
+          tar -zxf helm-v${HELM_VERSION}-linux-amd64.tar.gz
+          sudo mv linux-amd64/helm /usr/local/bin/helm
+          helm version
+          mkdir fixture
+        env:
+          HELM_VERSION: "3.4.2"
+
+      - name: Pack helm artifact
+        run: |
+          output=$(helm package ${CHART} --version ${CHART_VERSION})
+          echo $output
+        env:
+          CHART_VERSION: '0.0.0'
+          CHART: tyk-pro
+
+      - name: Push
+        id: push
+        uses: cloudsmith-io/action@master
+        with:
+          api-key: ${{ secrets.CLOUDSMITH_API_KEY }}
+          command: 'push'
+          format: 'helm'
+          owner: 'tyk'
+          repo: 'tyk-pro'
+          version: '0.0.0'
```

Comment:
Shouldn't this version correspond to the tag that was pushed?

Reply:
Hi @alephnull, thanks for chiming in! 

You're right, just didn't want to wait for a decision on how exactly we'll manage that to prevent a successful run. That means a release process to be created/maintained. 

Do you have an example mechanism I could base this one of? Not the actions per se, I've read your previous example and understand the implementation/triggers/variables - I mean the whole remaining organization, like maintainers, cadence, etc.

Reply:
Seems to me that you're in a position to define the process :)

Please add helm to https://tyktech.atlassian.net/wiki/spaces/EN/pages/449708061/Release+Engineering